crates/project_panel/src/project_panel.rs

```diff
@@ -1546,6 +1546,11 @@ impl ProjectPanel {
         destination_is_file: bool,
         cx: &mut ViewContext<Self>,
     ) {
+        // dropping on itself, fix #13093
```

Comment:
I don't think we need this comment, there's always `git blame` that we can use.
```suggestion
```

Reply:
Makes sense, I removed the comment.